samples/cpp/facedetect.cpp

```diff
@@ -202,18 +183,21 @@ void detectAndDraw( Mat& img, CascadeClassifier& cascade,
     int i = 0;
     double t = 0;
     vector<Rect> faces, faces2;
-    const static Scalar colors[] =  { CV_RGB(0,0,255),
-        CV_RGB(0,128,255),
-        CV_RGB(0,255,255),
-        CV_RGB(0,255,0),
-        CV_RGB(255,128,0),
-        CV_RGB(255,255,0),
-        CV_RGB(255,0,0),
-        CV_RGB(255,0,255)} ;
-    Mat gray, smallImg( cvRound (img.rows/scale), cvRound(img.cols/scale), CV_8UC1 );
+    const static Scalar colors[] =
+    {
+        Scalar(0,0,255),
```

Comment:
Also if this is just to grab different colors for different persons, why don't we use a randomn color generator here instead of predefined colors?